### ISSUE TITLE
Can't download TeamSiteFullWidth Sppkg

### ISSUE THREAD

**Issue by Alejandroo38** [devscopeninjas/dvs-spfx-extensions#1]
Hi there,

I would like to download the pluging TeamSiteFullWidth but I can't figure out where to download it ? How to install it into SharePoint ?

Thanks for your help,

Regards,

Alex

**Comment by CyberSauce** [comment]
Yes, some sort of directions would be beneficial in this process. I am trying to do this now with my classic team site.
